ts/Accessibility/KeyboardNavigation.ts

```diff
@@ -460,62 +460,71 @@ class KeyboardNavigation {
         element: DOMElementType
     ): void {
         const chart = this.chart,
-            keyboardNavigation = this;
-
-        this.eventProvider.addEvent(
-            element,
-            'focus',
-            function (ev: MouseEvent): void {
-                const e = ev || win.event,
-                    focusComesFromChart = (
-                        e.relatedTarget &&
-                        chart.container.contains(e.relatedTarget as any)
-                    ),
-                    comingInBackwards = !(
-                        focusComesFromChart || keyboardNavigation.exiting
-                    );
-
-                if (chart.focusElement) {
-                    delete chart.focusElement;
-                }
+            keyboardNavigation = this,
+            eventName = 'focus';
+
+        // Do not add event to the element if event is already added to prevent
+        // memory leak (#20329).
+        if (
+            (element as any).hcEvents &&
+            !(element as any).hcEvents[eventName]
+        ) {
+            this.eventProvider.addEvent(
+                element,
+                eventName,
```

Comment:
I think we should aim to fix this earlier, and not make an element that is already an exit anchor into an exit anchor again. So in `makeElementAnExitAnchor`, do a check if the element has already been assigned the class `highcharts-exit-anchor`, and if so, abort. Untested, but would be cleaner if that works.

Reply:
That one check if the element has already been assigned the class `highcharts-exit-anchor` didn't work well... I've changed solution a little bit. I am unhappy with `any`s, but it would easy to leave it as `any`.